### PR TITLE
Sberdoma-592 Contact overview pages with all details

### PR DESCRIPTION
**Pull request by SavelevMatthew**
image

Added contact overview page.
Done: 
1. basic contact information (organisations can only see their own contacts)
2. contact's tickets board (only tickets to current organization, clicking on ticket causing opening ticket page, clicking on more opens /ticket page with correct filters)

Not done for now:
1. Ability to delete contact (softDelete is not ready yet as @toplenboren said)
2. Ability to edit contact (will do it later)
3. No "owner" label, since we cannot acquire this information yet

### CODE REVIEW

File: apps/condo/pages/contact/[id]/index.tsx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    })
+
+    if (error || loading) {
+        return <LoadingOrErrorPage title={LoadingMessage} loading={loading} error={error ? ErrorMessage : null}/>
+    }
+    if (!contact) {
+        return <LoadingOrErrorPage title={ContactNotFoundTitle} loading={false} error={ContactNotFoundMessage}/>
+    }
+    // TODO (SavelevMatthew): more complex logic for enabling Contact editing?
+    const isContactEditable = true
```
Comment (Dimitreee) [inline apps/condo/pages/contact/[id]/index.tsx:82]: Use organizationEmployeeRole.canManageContacts field, @AntonAL can explain better

Comment (SavelevMatthew) [reply apps/condo/pages/contact/[id]/index.tsx]: done 👍

**Comment by SavelevMatthew** [comment]
Seems strange, but contact organization field if undefined...Need to fix that

**Comment by SavelevMatthew** [comment]
Fixed permission issue, open to review now :)

**Review by Dimitreee — APPROVED** [review]

**Review by dkoviazin — APPROVED** [review]

**Pull request merged**